neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -201,16 +234,17 @@ public virtual bool Verify(Snapshot snapshot, IEnumerable<Transaction> mempool)
                 return false;
             int size = Size;
             if (size > MaxTransactionSize) return false;
-            if (size > NativeContract.Policy.GetMaxLowPriorityTransactionSize(snapshot) && NetworkFee / size < NativeContract.Policy.GetFeePerByte(snapshot))
+            var feePerByte = NativeContract.Policy.GetFeePerByte(snapshot);
+            if (NetworkFee / size < feePerByte)
                 return false;
-            if (NativeContract.Policy.GetBlockedAccounts(snapshot).Contains(Sender))
+            if (NativeContract.Policy.GetBlockedAccounts(snapshot).Intersect(GetScriptHashesForVerifying(snapshot)).Count() > 0)
                 return false;
             BigInteger balance = NativeContract.GAS.BalanceOf(snapshot, Sender);
-            BigInteger fee = Gas + NetworkFee;
+            var fee = Gas + NetworkFee;
```

Comment:
Better leave `BigInteger`... more precise than `var`.

Reply:
I think that is long

Reply:
This is **exactly** the point. xD

Reply:
I think that is faster the comparison with `long`, but if you want, i can test it

Reply:
I think we should keep it `BigInteger`. Otherwise it may overflow in the next `Sum()`.